lib/nexpose/common.rb

```diff
@@ -105,22 +105,29 @@ class AdHocSchedule < APIObject
     # Force the scan to run during a blackout
     attr_accessor :force
 
-    def initialize(start, scan_template_id, max_duration, force = false)
+    def initialize(start, scan_template_id, max_duration = nil, force = nil)
       @start = start
       @scan_template_id = scan_template_id
-      @max_duration = max_duration
-      @force = force
+      @max_duration = max_duration if max_duration
+      @force = force if force
```

Comment:
It allows an admin user to create or run an adhoc schedule inside a blackout period, without it they are warned that they cannot create the schedule and advised to use the force